bokeh/core/tests/test_properties.py

```diff
@@ -32,14 +33,10 @@ class Foo(HasProps):
         self.assertEqual(f.s, None)
 
 
-        self.assertEqual(set(["x", "y", "z", "s"]), f.properties())
+        self.assertEqual(set(["x", "y", "z", "zz", "s"]), f.properties())
         with_defaults = f.properties_with_values(include_defaults=True)
-        del with_defaults['z'] # can't compare equality on the np array
-        self.assertDictEqual(dict(x=12, y="hello", s=None), with_defaults)
+        self.assertDictEqual(dict(x=12, y="hello", z=[1,2,3], zz={}, s=None), with_defaults)
         without_defaults = f.properties_with_values(include_defaults=False)
-        # the Array is in here because it's mutable
-        self.assertTrue('z' in without_defaults)
-        del without_defaults['z']
         self.assertDictEqual(dict(), without_defaults)
```

Comment:
we don't really use `Array` and might get rid of it. It's unstableness muddies this test.